migrations/20180713183206_remove_elevation_from_hxl_attributes.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class RemoveElevationFromHxlAttributes extends AbstractMigration
+{
+    /**
+     * Remove the 'elevation' HXL attribute.
+     * This attribute will be added back if we decide to implement elevation correctly
+     * in platform + csv
+     */
+    public function change()
+    {
+        $this->execute(
+            "DELETE FROM hxl_tag_attributes where attribute_id IN (
```

Comment:
I only deleted the reference so it doesn't appear. We are planning to add elevation at some point but we shouldn't see it in the list as an available option